src/direflow-components/Table/Table.js

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect, useState } from "react";
+import { Table as TableAntd } from "antd";
+import { requestEntity } from "./utils";
+import { capitalize } from "../../utils/utils_string";
+
+const Table = (props) => {
```

Comment:
a lo mejor podemos destructurar. Creo que quedaria algo asi:

{ displayEntities }

Reply:
I agree, also we can define a propType object for this component